### PR TITLE
Add a Bio for @GUPRAGHA

### PR DESCRIPTION
**Pull request by gupragha**
Pull request add bio for @gupragha as request number #13

### CODE REVIEW

File: docs/Raghav.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Raghav Gupta
+
+**Name:** Raghav Gupta
```
Comment (gupragha) [inline docs/Raghav.md]: Write short name here.

**Comment by 1992dheeraj** [comment]
Name size can be enhanced

File: docs/Raghav.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Raghav Gupta
+
+**Name:** Raghav
```
Comment (rthoma24) [inline docs/Raghav.md:3]: Full name needed

Comment (gupragha) [inline docs/Raghav.md:3]: Raghav Gupta :smiley:

**Pull request merged**